miner/worker.go

```diff
@@ -1238,13 +1238,13 @@ func (w *worker) commit(env *environment, interval func(), update bool, start ti
 		if interval != nil {
 			interval()
 		}
-
-		err := env.state.WaitPipeVerification()
-		if err != nil {
-			return err
-		}
-		env.state.CorrectAccountsRoot(w.chain.CurrentBlock().Root())
-
+		/*
```

Comment:
I will suggest that we disable the --pipecommit flag in this PR, in case someone still use it and come across big problem as we comment the code here.